tests/test_dse.py

```diff
@@ -258,9 +258,6 @@ class TestLifting(object):
         (['Eq(y.symbolic_max, g[0, x], implicit_dims=(t, x))',
          'Inc(h1[0, 0], 1, implicit_dims=(t, x, y))'],
          [6., 0., 0.]),
-        (['Eq(y.symbolic_max, g[0, x], implicit_dims=(t, x))',
```

Comment:
why dropping this?

Reply:
This example is wrong in 2 ways: #1695 #1696.
It should not be tested. The result we get is `by miracle` correct because the two issues manage to auto-cancel each other.

Reply:
OK